packages/context/src/binding.ts

```diff
@@ -873,6 +873,34 @@ export class Binding<T = BoundValue> extends EventEmitter {
   }
 }
 
+export declare interface Binding {
```

Comment:
We use declare here, but not for Context.

Reply:
What’s the impact on tsdocs?

Reply:
Personally, I don’t like to have the same name for class and interface.

Reply:
As far as I understand, we can simply use `export interface Binding`. The interface will be merged with the `typing` perspective of `Binding` class.

Reply:
> As far as I understand, we can simply use `export interface Binding`. The interface will be merged with the `typing` perspective of `Binding` class.

It works like a charm 👍 

I am not sure why I used `export declare interface` here when I am already using `export interface` in `src/context.ts` 🤷 

I discovered that `ContextView` is emitting events too, I'll add types for those events in this PR too.